roofit/roofit/src/RooPyBindFunction.cxx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Project: RooFit
+ * Authors:
+ *   Robin Syring, CERN 2024
+ *
+ * Copyright (c) 2024, CERN
+ *
+ * Redistribution and use in source and binary forms,
+ * with or without modification, are permitted according to the terms
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)
+ */
+
+#include "RooPyBindFunction.h"
+
+// ClassImp(RooPyBindFunction);
+
+
+/** \class RooPyBindFunction
+    \ingroup Roofit
+    \brief A RooFit class for wrapping python functions.
+
+This clsss provides the functionality to wrap arbitrary python functions in
+RooFit. 
+*/
+
```

Comment:
```suggestion

namespace RooFit {
namespace Detail {

```
The implementation needs to be in the same namespace (also think about the closing brackets at the end of the file).